README.md

```diff
@@ -1,33 +1,43 @@
-<p align="center"><img width="40%" src="https://hacktoberfest.digitalocean.com/assets/logo-hf19-full-10f3c000cea930c76acc1dedc516ea7118b95353220869a3051848e45ff1d656.svg"/></p>
+<p align="center"><img width="40%" src="https://hacktoberfest.com/_next/static/media/logo-hacktoberfest--horizontal.ebc5fdc8.svg"></p>
 
 # Hacktoberfest, 2019 with IEEE-VIT Student Branch
 
 SimpleMath-Flask is a Flask app that does math for you. This repository was created to help developers get started with Open Source during [Hacktoberfest](https://hacktoberfest.digitalocean.com/)
 > NOTE: Check out `issues` labelled as `Hacktoberfest` and start contributing!
 
-Made with :heart: by [IEEE-VIT](https://ieeevit.com)
+Made with :heart: by [IEEE-VIT](https://ieeevit.org/)
 
 Support Open Source by participating in [Hacktoberfest](https://hacktoberfest.digitalocean.com/) :yellow_heart:
 
 ## Getting started
 
-1. To setup this project, simply run the following command:
+1.Before setting up the project,Create A virtual environment in the project directory using the following command:
+```bash
+$ python3 -m venv .venv
+```
+2.The environment has been created just needs to be activated:
+```bash
+$ source .venv/bin/activate  ( for mac/linux users )  
```

Comment:
`(for mac/linux users)` and the other one maybe put that outside codeblocks?

Reply:
like??

Reply:
I've created a suggestion comment. Please check!